package.json

```diff
@@ -30,13 +34,16 @@
   },
   "homepage": "https://github.com/dfinity/hardware-wallet-cli#readme",
   "dependencies": {
-    "@dfinity/agent": "^0.14.0",
+    "@dfinity/agent": "^0.15.2",
+    "@dfinity/ledger": "next",
     "@dfinity/nns": "next",
-    "@dfinity/principal": "^0.14.0",
+    "@dfinity/principal": "^0.15.2",
+    "@dfinity/sns": "next",
     "@dfinity/utils": "next",
     "@ledgerhq/hw-transport-node-hid-noevents": "^6.3.0",
     "@ledgerhq/hw-transport-webhid": "^6.27.1",
     "@zondax/ledger-icp": "^0.6.0",
+    "buffer": "^6.0.3",
```

Comment:
out of curiosity, there was an error that requires adding `buffer`?

Reply:
Correct.

Reply:
Weird buffer is to polyfill for the browser 🤔. Maybe we can have a look one of these days

Reply:
yes, I want to take a quick look also